exareme2/node/config.toml

```diff
@@ -6,6 +6,7 @@ framework_log_level = "$FRAMEWORK_LOG_LEVEL"
 
 [privacy]
 minimum_row_count = 10
+protect_local_data = "$PROTECT_LOCAL_DATA"
```

Comment:
Update the README.md of node to also contain the PROTECT_LOCAL_DATA in the required env variables.